protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/FileBuilder.kt

```diff
@@ -36,9 +36,14 @@ open class FileBuilder(val namer: Namer = Namer.Standard, val supportMaps: Boole
             values + File.Type.Enum.Value(
                 number = value.number!!,
                 name = value.name!!,
-                kotlinValueName = namer.newEnumValueName(
+                kotlinValueClassName = namer.newEnumValueClassName(
```

Comment:
🙃 I think other parts of `pb-and-k` call strings used for class names `kotlinTypeName`

Reply:
👍